consensus/clique/clique.go

```diff
@@ -328,7 +328,7 @@ func (c *Clique) verifyCascadingFields(chain consensus.ChainHeaderReader, header
 	if parent == nil || parent.Number.Uint64() != number-1 || parent.Hash() != header.ParentHash {
 		return consensus.ErrUnknownAncestor
 	}
-	if parent.Time+c.config.Period > header.Time {
+	if header.Time < parent.Time {
```

Comment:
```suggestion
	if header.Time <= parent.Time {
```

Reply:
or is it OK to allow the same timestamp (looks viable).

Reply:
Yes, it is. We might end up having sub-second block times.

Reply:
it makes sense.